### ISSUE TITLE
Result Scoring

### ISSUE THREAD

**Issue by fschiettecatte** [ga4gh-discovery/ga4gh-case-discovery#12]
* Do we want scoring?
    * At what level, record, gene, variant, feature?
    * What range to use 0.00-1.00, 1-100, 1-1000?
    * Do we want to mandate use of specific scoring algorithms?
    * How do we represent scoring requested in the request, and scoring done in the response.

**Comment by Relequestual** [comment]
@harindra-a do we have any solid use cases and or algorithms that we can use to make an informed decision on this?

I assume that as we are not "matching" and in stead "searching", a score might not be so relevant, but I do not know.

**Comment by Relequestual** [comment]
I feel this should be a post 0.1.0 issue. Does anyone dissagree?

**Comment by harindra-a** [comment]
I agree with Ben here. This is either a post first-release issue, or not a need at all. I didn't comment on it earlier thinking it might be up for discussion at a meeting. 

We don't a have a use case for needing this (hence it was never in the drafts)

My feeling in general is that scoring is part of the "MME match" universe, and by empowering more access as we are trying to do now, a score is not needed.

I vote we put this for post October discussion and post some implementation testing to see if its needed. My feeling is it isn't.. We can talk about it more in next meeting too..

**Comment by fschiettecatte** [comment]
Discussions about this in V1 dragged in lots of additional issues, I am ok with punting this down the road.

**Comment by Relequestual** [comment]
Great.
As such, this will remain part of the post 0.1.0 milestone.
The expectation is, unless use cases are presented, this issue will be closed as "no need for further discussion" / "will not implement"
